### DISCUSSION TITLE
Authorization Denied when trying to export profiles

### DISCUSSION THREAD

**Discussion by adchst** [Keyfactor/ejbca-ce#519] [Q&A]
Hello,

I'm using the docker container of EJBCA 8.0.0. When I try to export for instance a certificate profile or an end entity profile, I get an Authorization Denied error message. How can I export the profiles? 

BR,
Christian

**Comment by primetomas** [comment]
Are you using it with certificate authentication for admin UI, or using the "simple" non-authenticated access?
I.e. how did you start the container? Non-authenticated access have some limitations.

**Reply by adchst** [reply]
Hello Tomas,

thanks for you reply. I used this command, to start EJBCA:
docker run -p 80:8080 -p 443:8443 -h localhost -e TLS_SETUP_ENABLED="simple" --mount source=[...],target=/mnt/persistent -it keyfactor/ejbca-ce:8.0.0

So I assume simple is not good? Any way I can repair that situation? I tried to start the container with TLS_SETUP_ENABLED="true", but I wasn't able to get the superadmin certificate. It looked to me, as if the login to EJBCA was still using simple mode, i.e. I wasn't queried for the superadmin certificate and the URL and password for the superadmin.p12 didn't show up in the EJBCA startup log.

**Reply by primetomas** [reply]
That is probably just generated during first installation. If you start an existing installation (with things in the database), it will not generate a new superadmin because it assumes you already have one. You can generate the superadmin manually however, and create a role to add it to, unless you want to make a re-installation.
